src/app/work-item/work-item-detail/work-item-detail.component.ts

```diff
@@ -324,7 +324,7 @@ export class WorkItemDetailComponent implements OnInit, AfterViewInit, OnDestroy
   getAllUsers(): Observable<any> {
     return Observable.combineLatest(
       this.userService.getUser(),
-      this.userService.getAllUsers()
+      this.workItemService.getCollaborators()
```

Comment:
We already have a collaborator service. You can use that instead creating a new **getCollaborators** in WI service. This is the file - **src/app/collaborator/collaborator.service.ts**